### PR TITLE
Fix issue where batch GCS renames were not issued

### PR DESCRIPTION
**Pull request by charlesccychen**

### CODE REVIEW

**Comment by charlesccychen** [comment]
R: @chamikaramj

File: sdks/python/apache_beam/io/fileio.py [hunk 1]
```diff
@@ -906,6 +909,8 @@ def _rename_batch(batch):
             logging.warning(('Exception in _rename_batch. src: %s, '
                              'dest: %s, err: %s'), src, dest, exception)
             exceptions.append(exception)
+        else:
+          logging.info('Rename successful: %s -> %s', src, dest)
```
Comment (chamikaramj) [inline sdks/python/apache_beam/io/fileio.py]: Probably should be logging.info to avoid spamming.

**Comment by chamikaramj** [comment]
LGTM

**Comment by charlesccychen** [comment]
Thanks!

R: @robertwb for merge

**Comment by robertwb** [comment]
@charlesccychen  please close.

**Pull request closed without merging**